### ISSUE TITLE
Add live region for important changes in document

### ISSUE THREAD

**Issue by nvdaes** [daisy/ace-gui#70]
After checking an ePUB, it would be useful to notify screen readers about messages.
I have created a branch with the following additions:

- Make the div for rendered messages aria-live="polite" with role="complementary" so that it can be located (other landmarks could be added too).
- After messages are rendered with the div, clear the message list setting its length to 0, so that the div is announced just when new messages are placed there, without being too verbose or keeping a lot of previous info if rerun Ace is performed..
I will create a PR, but you may want discuss some aspects or code improvements here.

**Comment by danielweck** [comment]
Pull Request: https://github.com/daisy/ace-gui/pull/71

**Comment by nvdaes** [comment]
Also, it is possible to add an accessible name to the landmark. But sometimes accessible names in this context make landmarks too verbose. If not needed and the document has just this complementary landmark when available, I would add a main landmark for all documents, and if possible a contentinfo too. I think this should be addressed as a whole, but for me this live region for messages, and even clearing them in each update after render, would be a good improvement for screen readers.

**Comment by nvdaes** [comment]
Also, Pull Request: #73

**Comment by nvdaes** [comment]
I have added aria-live="polite" also for the report table. For me this impreves the user experience with a screen reader, since when I press the buttons to order the table values, for example by name, accessibility, etc., at least this is notified and we can then explore the table. Also, I can use the filter by fields in browse mode, though this is not the ideal solution.
I have join tables and messages in this issue and the aame pull request.

**Comment by danielweck** [comment]
It is really great that you are able to

**Comment by danielweck** [comment]
It is really great that you are able to test your own code modifications. Thank you so much. I will review your Pull Requests as soon as possible.

**Comment by nvdaes** [comment]
Thanks for reviewing. For me these changes are very important, since I can know if the table report has or not changed using filter by and reorder options.
Also, I know when Ace is running and when the report has been load.
I have added a main landmark and we may add also a landmark with some role like menu or something, for the sidebar and the settings element, all together. I'm translating axe-core and rules are that all document elements are contained in a landmark and that this should be unique. Also, certain landmarks like main or complementary should be at top level, i.e., not contained in other landmark. It would be nice if Ace could follow its own rules.
Also a rule refers to lang element. Hope you can review the corresponding PR also send by me.
Thanks for all your work.
